src/misc/index.ts

```diff
@@ -0,0 +1,57 @@
+import { generateValidator, JSONSchema, ValidateFunction } from ".."
+
+/**
+ * Color3 is a data type that describes a color using R, G and B components
+ * @alpha 
+ */
+export type Color3 = { r: number; g: number; b: number }
+export namespace Color3 {
+  export const schema: JSONSchema<Color3> = {
+    type: 'object',
+    required: ['r', 'g', 'b'],
+    properties: {
+      r: {
+        type: 'number',
+        minimum: 0,
+        maximum: 1
```

Comment:
is this correct? shouldn't it be 255?

Reply:
as I can see from the samples, it is a floating number in range [0,1]